src/test/ui/type-alias-enum-variants/enum-variant-priority-higher-than-other-inherent.stderr

```diff
@@ -5,12 +5,6 @@ LL |     <E>::V();
    |     ^^^^^^-- supplied 0 arguments
    |     |
    |     expected 1 argument
-   |
-note: tuple variant defined here
-  --> $DIR/enum-variant-priority-higher-than-other-inherent.rs:5:5
-   |
-LL |     V(u8)
-   |     ^^^^^
```

Comment:
Somewhat surprised by this change 👀

Reply:
Yes, it's a bit unfortunate side effect of looking up identifiers via a new query - `def_ident_span` - that I haven't been able to re-implement for tuple variants. Technically reviving this one should be as easy as improving that query.